docs/install.rst

```diff
@@ -139,6 +139,8 @@ Requirements
 
 - `PyERFA`_ |minimum_pyerfa_version| or later
 
+- `PyYAML <https://pyyaml.org>`_ |minimum_pyyaml_version| or later
+
 ``astropy`` also depends on a number of other packages for optional features.
 The following three are particularly recommended:
```

Comment:
Sorry, now the "three" here should become "two" (or maybe just remove the number...)

Reply:
Agreed. It's now "The following are particularly recommended:"